### ISSUE TITLE
Broken display of options with icons

### ISSUE THREAD

**Issue by dragomano** [sa-si-dev/virtual-select#311]
Source code
```html
		<div id="tags" name="tags"></div>
		<script>
			VirtualSelect.init({
				ele: "#tags",
				dropboxWrapper: "body",
				maxWidth: "100%",
				multiple: true,
				search: true,
				markSearchResults: true,
				showValueAsTags: true,
				allowNewOption: false,
				showSelectedOptionsFirst: true,
				placeholder: "Select tags or add new",
				noSearchResultsText: "Sorry, no matches were found",
				searchPlaceholderText: "Search",
				noOptionsText: "No tags yet",
				clearButtonText: "Remove",
				maxValues: 10,
				options: [{"label":"<i class=\"fas fa-snowman\" aria-hidden=\"true\"><\/i> Cat","value":2},{"label":"<i class=\"fas fa-cheese\" aria-hidden=\"true\"><\/i> Tag","value":1}],
				selectedValue: [2,1]
			});
		</script>
```

v1.0.40
![sshot-6](https://github.com/sa-si-dev/virtual-select/assets/229402/146e027d-4929-4c54-b097-60eb27c00602)

v1.0.41
![sshot-4](https://github.com/sa-si-dev/virtual-select/assets/229402/4c3b17e7-3c1c-47ce-a760-b59bd835b401)

Did something get broken somewhere?

**Comment by gnbm** [comment]
@dragomano Seems like a regression issue while using `multiple: true`

**Comment by gnbm** [comment]
@dragomano Opened a [PR](https://github.com/sa-si-dev/virtual-select/pull/315) that includes a fix for this. Can you please review it?

**Comment by dragomano** [comment]
Confirm, the fix resolved this problem.

**Comment by gnbm** [comment]
Fixed in v1.0.42
